src/lib/Components/LocationMap/index.tsx

```diff
@@ -33,6 +33,63 @@ interface Props {
   partnerName?: string
 }
 
+export const cityAndPostalCode = (city, postalCode) => {
+  if (city && postalCode) {
+    return city + ", " + postalCode
+  } else if (city) {
+    return city
+  } else if (postalCode) {
+    return postalCode
+  }
+}
+
+export const tappedOnMap = (address, summary, partnerName, city, postalCode, lat, lng) => {
```

Comment:
Long argument lists like this scare me a little 😅Good candidate for the single arg object pattern I reckon.